docs: add forceMount prop (#134)
pacocoursey/cmdk: README.md

</Command.Item>
```

> You can force an item to always render, regardless of the search, with `forceMount` prop

Reviewer: ```suggestion
You can force an item to always render, regardless of filtering, by passing the `forceMount` prop.
```

Groups will not unmount from the DOM, rather the `hidden` attribute is applied to hide it from view. This may be relevant in your styling.

> And you can force a group to always render with `forceMount` prop

Reviewer: ```suggestion
You can force a group to always render, regardless of filtering, by passing the `forceMount` prop.
```
